ui/app/src/services/marketplace.ts

```diff
@@ -17,7 +17,7 @@
 import { get, post } from "../utils/ajax";
 import { MarketplaceSite } from "../models/Site";
 
-export function fetchBlueprints() {
+export function fetchBlueprints(showIncompatible: boolean) {    //TODO: API still not updated with the param
```

Comment:
Make the function receive a `options` object with one of the options being `showIncompatible`. Include also type and limit (i.e. all of the currently hard coded params). Assign defaults to all so it keeps working the same way. Make the argument optional. 

Find out from @jdrossl what the API will look like so you can move forward with using the value.

Reply:
@rart API still not defined, talked to Jose Ross and decided to use the `showIncompatible` param. Will add to docs